src/nora/components/NoraButton/NoraButton.module.scss

```diff
@@ -0,0 +1,17 @@
+.Button {
+  font-weight: 500;
+  font-size: 13px;
+  height: var(--Space-32);
+  line-height: var(--Space-32);
+  letter-spacing: 0.015em;
+  color: white;
+  background: var(--NoraBlue-400);
+  border-radius: 3px;
+  text-align: center;
+  display: inline-block;
+}
```

Comment:
On my branch for the Notes textarea that this will soon replace, I'm doing:

```scss
.Submit[disabled] {
  opacity: 0.65;
}
```

so you should probably do here:

```
.Button[disabled] {
  opacity: 0.65;
}
```

I copied `.65` from bootstrap but I could be persuaded to use `.7` if we prefer :)

Reply:
hm, should these styles be passed in through the className instead? or does that not work